qiskit_nature/properties/second_quantization/vibrational/bases/vibrational_basis.py

```diff
@@ -39,7 +39,7 @@ def __init__(
 
     @property
     def num_modals_per_mode(self) -> List[int]:
-        """Returns the num_modals_per_mode."""
+        """Returns the number of modals per mode."""
```

Comment:
In the class documentation a few lines above this

> Refer to the documentation of ``qiskit_nature.properties.vibrational.integrals`` for more details.

This would be nice to have as a link to the module since the user for when the user is already reading the docs